dist/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Weather API</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="">
+  </head>
+  <body>
+    <h1>See your city's weather</h1>
```

Comment:
- [ ] Your application should also include a toggle button that defines if the temperature will be shown in Celcius or Fahrenheit, please add the toggle here and the necessary attached functions in the JS files..